src/components/PickupBox/PickupBox.jsx

```diff
@@ -83,8 +84,68 @@ const PickupBox = ({
             {/* Box details */}
             <AccordionPanel className={styles['accordion-panel']} pb={4}>
               <div className={styles['box-details']}>
-                {picture !== null && (
-                  <img src={picture} alt="" className={styles['pickup-image-corners']} />
+                {(!(status === 'evaluated') || !(imageStatus === 'rejected')) && picture && (
```

Comment:
Could we change this syntax to ((status !== 'evaluated') || (imageStatus !== 'rejected'))? Correct me if I'm wrong, but this should serve the same logic and helps with readability.